db/models/index.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const fs        = require("fs");
+const path      = require("path");
+const Sequelize = require("sequelize");
+const basename  = path.basename(__filename);
+const env       = process.env.NODE_ENV || "development"; // eslint-disable-line no-process-env
+const config    = require(__dirname + "/../config/config.json")[env];
+const db        = {};
+
+let sequelize;
+
+if (config.use_env_constiable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config); // eslint-disable-line no-process-env
```

Comment:
Instead of `process.env`, not sure if we could use `AppConstants[config.use_env_variable]` wrapper. Or just `AppConstants.DATABASE_URL` directly, since all 3 envs seem to use the same key.